trl/trainer/reward_trainer.py

```diff
@@ -244,6 +244,9 @@ def compute_loss(
         else:
             loss = -nn.functional.logsigmoid(rewards_chosen - rewards_rejected).mean()
 
+        if self.args.center_rewards_coefficient is not None:
+            loss += self.args.center_rewards_coefficient * nn.functional.mse_loss(rewards_chosen, rewards_rejected)
```

Comment:
```suggestion
            loss += self.args.center_rewards_coefficient * torch.mean((rewards_chosen, rewards_rejected) ** 2)
```

Reply:
following https://github.com/huggingface/trl/issues/1931#issuecomment-2294902065, is it correct?

Reply:
@qgallouedec your implementation is correct.